src/home/home.py

```diff
@@ -1,5 +1,4 @@
 import requests
-import datetime
```

Comment:
Should we avoid the same names for these subfolders? maybe `homepage/home.py` is a better?

Reply:
the double names don't seem so bad, all of the subfolders have that pattern

and it made [this change I made](https://github.com/NYCPlanning/data-engineering-qaqc/blob/004654295315b63ad06e4276939d8e21a8cd4bc9/index.py#L32-L34) in `index.py` relatively simple